governance-exercises/CivicSage Exercises/Ex01/2. forum.md

```diff
@@ -0,0 +1,10 @@
+## Draft any forum responses required or implied by your course of action.
+
+The Governance Facilitators propose structuring the changes to Spork listed by @Alice and @Faraz in three individual polls:
```

Comment:
Keep in mind that there will be other Governance Facilitators. It's advisable to avoid speaking on their behalf. You could phrase it as "Endgame Edge, acting as the Governance Facilitator, ..."